implementations/rust/.gitignore

```diff
@@ -1,2 +1,4 @@
 # Cargo stores the output of a build into the target directory.
 target
+/Cargo.toml
+/Cargo.lock
```

Comment:
could you move this change to a separate commit.

Reply:
@mrinalwadhwa fixed